Sources/Core/Client/Channel/Channel+Events.swift

```diff
@@ -15,15 +15,15 @@ public extension Channel {
     /// Observe events with a given event type and channel.
     /// - Parameter eventType: an event type.
     /// - Returns: an observable channel events.
-    func onEvent(eventType: EventType, _ onNext: @escaping Client.Completion<StreamChatClient.Event>) -> Subscription {
```

Comment:
Did you have to change these names? 😂 I'll change them anyway soon?

Reply:
I know, that's why I asked you to update Core later.

Reply:
But why did you need to rename them?

Reply:
I refactored `rx.connection` it wasn't needed `onConnect` and `connection` on the same time.
I'm not sure that we need `connected` with implementation on LLC. I want to check that after your PR.